src/ConfigManager.cpp

```diff
@@ -342,6 +337,45 @@ void ConfigManager::createBaseWebServer() {
     server->onNotFound(std::bind(&ConfigManager::handleNotFound, this));
 }
 
+
+void ConfigManager::clearWifiSettings(bool reboot) {
+    char ssid[SSID_LENGTH];
+    char password[PASSWORD_LENGTH];
+    memset(ssid, NULL, SSID_LENGTH);
+    memset(password, NULL, PASSWORD_LENGTH);
+    storeWifiSettings(ssid, password, true);
+
+    if (reboot) {
+        ESP.restart();
+    }
+}
+
+void ConfigManager::storeWifiSettings(String ssid, String password, bool reset) {
```

Comment:
This is not wrong, but bool parameters like this are not super great. Look at the usage `storeWifiSettings(ssid, password, false)`, the `false` is not descriptive. Perhaps `(char[WIFI_OFFSET] magicBytes, String ssid, String password)` would be easier to use?

Reply:
I didnt want to pass the magicBytes (set or empty) -- but this will work. Can change

Reply:
Yeah, i totally get that. Just from a usage PoV, no one ever know what the `true` or `false` is for,

Reply:
This proved more challenging -- almost gave up -- but I have it working, so confused however